### DISCUSSION TITLE
When are changed file-dates visible in the GUI?

### DISCUSSION THREAD

**Discussion by DJF3** [immich-app/immich#8260] [Q&A] [answered]
Team,
Just found out that there's 100s of images and videos that have the wrong file **modified**-date and **created**-date.
After creating a shell script (on Macos) that updates the date modified/created for each file based on the EXIF data. 

Then, in Immich, I selected each image and clicked "Refresh Metadata".
However, under Info/Details the old dates are still shown.

_What is the best way to get Immich to see and use the updated file dates?_

**Comment by schuhbacca** [comment, marked as answer]
The best way, is to download the images that you want to fix, fix them, then re-upload them. The file hash is based off of the extracted data. If you change a file directly the hash will no longer match and can cause problems. 

Things such as duplicate detection won't work, and probably sinking of the local and remote assets on a mobile device.

**Reply by DJF3** [reply]
Uh oh..
I have corrected 1000s of file dates :-) 
Thanks for the info!
